sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -164,3 +164,16 @@ public partial class SasQueryParameters
         public override string ToString() { throw null; }
     }
 }
+namespace Azure.Storage.Shared
+{
+    public partial class StorageBearerTokenChallengeAuthorizationPolicy : Azure.Core.Pipeline.BearerTokenAuthenticationPolicy
```

Comment:
Why is this public?  Can't we use this automagically if customers want tenant discovery?

Reply:
Yes, I meant to mark this as internal.